### PR TITLE
Default support for ES queries

### PR DESCRIPTION
**Pull request by calina-c**
## Description

Support native ES search by default. Remove stale routes.

## Is this PR related with an open issue?

Related to Issue #372

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change) -> need to check all usages in market repo
 
## Checklist:

- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

**Comment by calina-c** [comment]
I focused on implementing this with minimal changes to the market flow. But sorting, pagination etc. need more refining.

**Comment by mihaisc** [comment]
This look good from my point of view.

**Comment by kremalicious** [comment]
cool, one way of searching down. Wondering if we should also deal with `GET /ddo/query`. It seems there's also a `text` parameter on `POST /ddo/query`. Are they both doing the same thing, and returning the same thing? If so we could remove `GET /ddo/query`, no? If we keep it it should be named more REST-like: `GET /ddo/text`

**Comment by mihaisc** [comment]
We don't use that in the market right? From my point of view we can remove that. That search can be done with an es query in post.

**Comment by calina-c** [comment]
> cool, one way of searching down. Wondering if we should also deal with `GET /ddo/query`. It seems there's also a `text` parameter on `POST /ddo/query`. Are they both doing the same thing, and returning the same thing? If so we could remove `GET /ddo/query`, no? If we keep it it should be named more REST-like: `GET /ddo/text`?

The GET request is a shortcut to just searching by text.
The POST request is an ES query.

We can keep both, one or none. Just let me know. I'm always happy to delete code (and not break anything).

File: aquarius/app/assets.py [hunk 1]
```diff
@@ -200,136 +200,7 @@ def get_assets_names():
...
-    response = make_paginate_response(query_result, search_model)
-    return Response(
-        json.dumps(response, default=datetime_converter),
-        200,
-        content_type="application/json",
-    )
-
-
-@assets.route("/ddo/esquery", methods=["POST"])
 def es_query_ddo():
```
Comment (ssallam) [inline aquarius/app/assets.py:204]: rename to `query_ddo`

File: aquarius/app/dao.py [hunk 2]
```diff
@@ -84,33 +84,28 @@ def is_listed(services):
...
             sort = [{"_id": "asc"}]
 
-        if text:
-            sort = [{"_score": "desc"}] + sort
-            if isinstance(text, str):
-                text = [text]
-
-            text = [t.strip() for t in text]
-            text = [t.replace("did:op:", "0x") for t in text if t]
-
```
Comment (ssallam) [inline aquarius/app/dao.py:110]: is text search deprecated? does the query alone support the existing use cases in the market?

Comment (calina-c) [reply aquarius/app/dao.py:110]: Yes, the query can have text inside it, market asked to remove the shortcut

**Review by ssallam — APPROVED** [review]
looks good.

**Pull request merged**